src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -339,6 +340,13 @@ public AgentSettings LoadSettings()
                 var serviceControlManager = HostContext.GetService<IWindowsServiceControlManager>();
                 serviceControlManager.ConfigureService(agentSettings, command);
             }
+            //This will be enabled with AutoLogon code changes are tested
```

Comment:
please let me know if this doesnt answer your question.

---
In reply to: [117705064](https://github.com/Microsoft/vsts-agent/pull/978#discussion_r117705064) [](ancestors = 117705064,117651381)